pkg/controller/plan/adapter/vsphere/client.go

```diff
@@ -41,6 +48,7 @@ type Client struct {
 
 // Create a VM snapshot and return its ID.
 func (r *Client) CreateSnapshot(vmRef ref.Ref) (id string, err error) {
+	log.V(1).Info("Creating snapshot", "vmRef", vmRef)
```

Comment:
why not using r.Log..?

Reply:
yeah, for consistency with other adapters, better to use the logger within the `Context`

Reply:
I keep forgetting about that one. Done.